### PR TITLE
Update _data file from Meetup API to create static events page

### PR DESCRIPTION
**Pull request by edgargigaramirez**
Using [meetup_api](https://www.meetup.com/meetup_api/docs/) is possible to generate a `_data/` file and update events page with new events.

screen shot 2018-09-10 at 11 23 06 pm
screen shot 2018-09-10 at 11 23 12 pm

### CODE REVIEW

File: meetup.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+rm _data/meetup/events.json
+curl -o output.json "https://api.meetup.com/openuptown/events?key={YOUR_API_KEY}}"
+cat output.json | jq '.[95:100]' >> _data/meetup/events.json
```
Comment (edgargigaramirez) [inline meetup.sh]: This needs to be improved. At this moment there are 100 events returned by the API, what I am trying to do is to return the latest 5. I can either use a variable to store the number of events and simply subtract 5 from it.

I'd like to hear better approaches.

**Comment by edgargigaramirez** [comment]
Please take a look to the `README.md` for more details about how it's getting updated the `_data` files.

File: meetup.sh (hunk 1 above)
Comment (Ryan-Koch) [reply meetup.sh]: This should be pretty easy to remedy. The */events API end point for meetup allows you to use a page argument that takes a number that represents the number of events you want it to respond with. E.g. &page=<some number>

Comment (edgargigaramirez) [reply meetup.sh]: Good call.  Let me check it out and I’ll change it.

**Pull request merged**